apps/src/applab/designElements/EnumPropertyRow.jsx

```diff
@@ -0,0 +1,38 @@
+var rowStyle = require('./rowStyle');
+
+var EnumPropertyRow = React.createClass({
+  propTypes: {
+    initialValue: React.PropTypes.string.isRequired,
+    options: React.PropTypes.arrayOf(React.PropTypes.string).isRequired,
+    handleChange: React.PropTypes.func
+  },
+
+  getInitialState: function () {
+    return {
+      selectedValue: this.props.initialValue
+    };
+  },
+
+  handleChange: function (event) {
+    this.props.handleChange(event.target.value);
+    this.setState({selectedValue: event.target.value});
+  },
+
+  render: function () {
+    let options = this.props.options.map(
+      (option, index) => <option key={index} value={option}>{option}</option>
+    );
```

Comment:
Wish I didn't have to say this, but we're not allowing ES6 yet :frowning: sorry!